apps/percona/chaos/drain_node/run_litmus_test.yml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: node-failure-drain-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      labels:
+        name: node-failure-drain
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      #nodeSelector:
+      #  kubernetes.io/hostname: 
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: Always
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            #value: log_plays
+            #value: actionable
+            value: default
+
+          - name: APP_NAMESPACE
+            value: app-percona-ns 
+
+          - name: APP_LABEL
+            value: 'name=percona'
+
+          - name: LIVENESS_APP_LABEL
+            value: "liveness=node-chaos"
+
+          - name: LIVENESS_APP_NAMESPACE
+            value: "litmus"
+
+        command: ["/bin/bash"]
+        args: ["-c", "ansible-playbook ./percona/chaos/drain_node/test.yml -i /etc/ansible/hosts -vv; exit 0"]
+        volumeMounts:
+          - name: logs
+            mountPath: /var/log/ansible
+        tty: true
+
+      - name: logger
+        image: openebs/logger
+        env:
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          # spec.volumes is not supported via downward API
+          - name: MY_POD_HOSTPATH
+            value: /mnt/chaos/node-failure-drain
+        command: ["/bin/bash"]
+        args: ["-c", "./logger.sh -d ansibletest -r maya,openebs,pvc,percona; exit 0"]
+        volumeMounts:
+          - name: kubeconfig
+            mountPath: /root/admin.conf
+            subPath: admin.conf
+          - name: logs
+            mountPath: /mnt
+        tty: true
+      volumes:
+        - name: kubeconfig
+          configMap:
+            name: kubeconfig
+        - name: logs
+          hostPath:
+            path: /mnt/chaos/node-failure-drain
+            type: ""
+
+      tolerations:
```

Comment:
any particular reason in placing tolerations here @dargasudarshan ?

Reply:
No particular reason @ksatchit Just followed k8s docs example.

Reply:
```
kubectl taint node gke-e2e-cstor-dev-1-default-pool-224a4771-gdmk infra-aid=observer:NoSchedule                 
node "gke-e2e-cstor-dev-1-default-pool-224a4771-gdmk" tainted
sudarshan_darga@cloudshell:~/pr/litmus/apps/percona/liveness (strong-eon-153112)$ watch kubectl get pods -n app-percona-ns -o wide                                                                
sudarshan_darga@cloudshell:~/pr/litmus/apps/percona/liveness (strong-eon-153112)$ cd ..
sudarshan_darga@cloudshell:~/pr/litmus/apps/percona (strong-eon-153112)$ cd chaos/drain_node/
sudarshan_darga@cloudshell:~/pr/litmus/apps/percona/chaos/drain_node (strong-eon-153112)$ ls
run_litmus_test.yml  test_vars.yml  test.yml
sudarshan_darga@cloudshell:~/pr/litmus/apps/percona/chaos/drain_node (strong-eon-153112)$ vi run_litmus_test.yml
sudarshan_darga@cloudshell:~/pr/litmus/apps/percona/chaos/drain_node (strong-eon-153112)$ kubectl create -f run_litmus_test.yml
job.batch "node-failure-drain-cl57b" created
sudarshan_darga@cloudshell:~/pr/litmus/apps/percona/chaos/drain_node (strong-eon-153112)$ kubectl get pods -n litmus -o wide
NAME                             READY     STATUS              RESTARTS   AGE       IP            NODE
litmus-percona-k8kkj-w6bn6       0/1       Completed           0          8m        10.20.22.14   gke-e2e-cstor-dev-1-default-pool-224a4771-jq6g
node-failure-drain-cl57b-6kbjr   0/2       ContainerCreating   0          10s       <none>        gke-e2e-cstor-dev-1-default-pool-224a4771-gdmk
```